index.html

```diff
@@ -27,7 +33,7 @@
         <section class="hearder-section">
           <div class="hearders">
             <div class="title">
-              <h1 class="h-title">I’m TATOU. Glad to see you!</h1>
+              <h1 class="h-title">I’m ROBERTO. Glad to see you!</h1>
```

Comment:
- [ OPTIONAL ] In your headline section, this heading is not properly styled as seen in the Figma design and this is also not giving your Portfolio an overall good look and feel. So even tho this is an optional suggestion, I will suggest you fix it to make your project look more similar to the Figma design and more appealing to the end user😉  
- Your Design
<img width="1198" alt="Screen Shot 2023-05-13 at 10 13 56 AM" src="https://github.com/tatouroberto/portfolio-project/assets/102699958/d21828b9-4912-4e25-908e-87be1e21878a">

- The Figma Design
<img width="1277" alt="Screen Shot 2023-05-13 at 11 06 04 AM" src="https://github.com/tatouroberto/portfolio-project/assets/102699958/f8dc62e0-b59f-4219-8ce7-fe3410b309d2">